script.js

```diff
@@ -0,0 +1,32 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const compVal = ['rock','paper','scissors'];
+let humanVal;
+
+function computerPlay() {   
+    let rand = Math.floor(Math.random()*compVal.length);
+    return compVal[rand];  
+}
+
+function humanPlay() {
+    let humanVal = prompt("Enter value:");
+    return humanVal;  
+}
+
+function roundPlay(compVal, humanVal) {
+    if (compVal == humanVal) {
+        return("draw");
+    } else if ((compVal === "rock" && humanVal === "paper")||
+               (compVal === "paper" && humanVal === "scissors")||
+               (compVal === "scissors" && humanVal === "rock")) {
```

Comment:
You're using strings here, like `"rock"`, but you already have these in the constants above
https://github.com/Cybi1/rock-paper-scissors/blob/b15c25d31eaa7c07783f6121fb63e2cce7571fab/script.js#L1-L3
So use those constants, instead of strings here

Reply:
fixed